src/tests/verification_mock/mod.rs

```diff
@@ -94,12 +99,13 @@ pub(super) fn verification_without_mock_root() {
         std::time::SystemTime::now(),
     );
 
-    assert_eq!(
-        result.map(|_| ()),
-        Err(TlsError::InvalidCertificateData(String::from(
-            error_messages::UNKNOWN_CERT
-        )))
-    );
+    let res = result.map(|_| ());
+    assert!(matches!(
+        res,
+        Err(TlsError::InvalidCertificate(
+            CertificateError::UnknownIssuer
+        ))
+    ));
```

Comment:
With the return of `PartialEq`, this change can be reverted right?

Reply:
Good catch. I missed this in my cleanup. Will adjust.

Reply:
Fixed!